### ISSUE TITLE
Cypress CLI should treat `-h` `--help` and `help` as valid command

### ISSUE THREAD

**Issue by bahmutov** [cypress-io/cypress#645]
@bahmutov commented on [Wed Sep 13 2017](https://github.com/cypress-io/cypress-monorepo/issues/463)

Currently it treats it as unknown command
```
$ bin/cypress help
Unknown command "help"

  Usage: cypress [options] [command]


  Commands:

    version          Prints Cypress version
    run [options]    Runs Cypress tests from the CLI without the GUI
    open [options]   Opens Cypress in the interactive GUI.
    install          Installs the Cypress executable matching this package's version
    verify           Verifies that Cypress is installed correctly and executable

  Options:

    -h, --help     output usage information
    -v, --version  Prints Cypress version

$ bin/cypress -h
Unknown command "-h"

  Usage: cypress [options] [command]


  Commands:

    version          Prints Cypress version
    run [options]    Runs Cypress tests from the CLI without the GUI
    open [options]   Opens Cypress in the interactive GUI.
    install          Installs the Cypress executable matching this package's version
    verify           Verifies that Cypress is installed correctly and executable

  Options:

    -h, --help     output usage information
    -v, --version  Prints Cypress version

$ bin/cypress --help
Unknown command "--help"

  Usage: cypress [options] [command]


  Commands:

    version          Prints Cypress version
    run [options]    Runs Cypress tests from the CLI without the GUI
    open [options]   Opens Cypress in the interactive GUI.
    install          Installs the Cypress executable matching this package's version
    verify           Verifies that Cypress is installed correctly and executable

  Options:

    -h, --help     output usage information
    -v, --version  Prints Cypress version
```

**Comment by brian-mann** [comment]
The code for this is merged, but it has not been released yet.

**Comment by brian-mann** [comment]
Fixed in [`0.20.1`](https://docs.cypress.io/guides/references/changelog.html#0-20-1).
